python/lsst/meas/mosaic/checkMosaicTask.py

```diff
@@ -56,12 +56,13 @@ def makeDiffPosFlux(self, allMat, allSource, wcsDic, calibDic, ffpDic):
                     iexp = ss[j].getExp()
                     ichip = ss[j].getChip()
                     x, y = ss[j].getX(), ss[j].getY()
-                    ra, dec = wcsDic[iexp][ichip].pixelToSky(x, y).getPosition()
+                    wcs = wcsDic[iexp][ichip]
+                    ra, dec = wcs.pixelToSky(x, y).getPosition()
                     dx_m.append((ra - ra_cat) * 3600)
                     dy_m.append((dec - dec_cat) * 3600)
                     mag = 2.5*math.log10(calibDic[iexp][ichip].getFluxMag0()[0]/ss[j].getFlux())
                     mcor = ffpDic[iexp][ichip].eval(x,y)
-                    jcor = -2.5*math.log10(wcsDic[iexp][ichip].pixArea(afwGeom.Point2D(x, y)) / wcsDic[iexp][ichip].pixelScale().asDegrees()**2)
+                    jcor = -2.5*math.log10(measMosaic.computeJacobian(wcs, afwGeom.Point2D(x, y)))
```

Comment:
Should these have been part of the previous commit?  (or is there some reason that wouldn't have worked?)